moduleroot/spec/spec_helper.rb.erb

```diff
@@ -31,6 +31,38 @@ default_fact_files.each do |f|
   end
 end
 
+# @return [String] - the path to the fixtures directory
+def fixtures_dir
+  @fixtures_dir ||= File.join(__dir__, 'fixtures')
+end
+
+# @return [String] = the path the directory of external facterdb facts
+# Most users will likely not have a custom facterdb_facts git repo defined
+# but if they do then can add it to the fixtures file and name it facterdb_facts.
+# See https://github.com/camptocamp/facterdb#supplying-custom-external-facts
+def mock_facts
```

Comment:
I think this (and the environment variable setting below) would be better suited to be part of `rspec-puppet-facts`.